@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf

```diff
@@ -122,29 +127,43 @@ if (_velXY < 12.35) then {  //must be less than ETL
         setCamShakeParams [0.0, 0.5, 0.0, 0.0, true];
         addCamShake       [2.5, 1, 5];
         enableCamShake false;
+
+        SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
+        SetCustomSoundController[_heli, "CustomSoundController4", 1.8];
     };
     //2933 fpm to 3867 
     if (_velZ <= -14.89 && _velZ > -19.64) then {
         enableCamShake true;
         setCamShakeParams [0.0, 0.5, 0.0, 0.5, true];
         addCamShake       [3, 1, 5.5];
         enableCamShake false;
+
+        SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
+        SetCustomSoundController[_heli, "CustomSoundController4", 1.8];
     };
     //3867fpm to 4800 fpm
     if (_velZ <= -19.64 && _velZ > -24.384) then {
         enableCamShake true;
         setCamShakeParams [0.0, 0.75, 0.0, 0.75, true];
         addCamShake       [3.5, 1, 6.0];
         enableCamShake false;
+
+        SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
+        SetCustomSoundController[_heli, "CustomSoundController4", 1.8];
     };
     //> 4800fpm
     if (_velZ < -24.384) then {
         enableCamShake true;
         setCamShakeParams [0.0, 1.0, 0.0, 2.0, true];
         addCamShake       [4.0, 1, 6.5];
         enableCamShake false;
+
+        SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
```

Comment:
Why have we started at 3, are there 2 more I can't see in the diff or are they reserved?

Reply:
I copied Arens code and didn't want to break it....